### DISCUSSION TITLE
Google search settings

### DISCUSSION THREAD

**Discussion by maccagnan** [qutebrowser/qutebrowser#8129] [General] [answered]
Hello,
if I set my search engine default settings to:
{"DEFAULT": "https://duckduckgo.com/?q={}"}
and then type a word in the :open command bar, the duckduckgo page shows directly the search results.
But if I set my search engine settings to:
{"DEFAULT": "https://google.com/?q={}"}
then the imput term appears in Google search tray, and I have to press I+enter to get the results.
How do I get google to search directly for the imputted term?
I know this can be done because my previous QB setup worked that way, but after trying several possible configurations I can't make it work.

**Comment by toofar** [comment, marked as answer]
Use `https://www.google.com/search?q={}`. So put the `search` in there before `?q`.

**Comment by maccagnan** [comment]
Awesome! Thanks a lot for the prompt reply.

**Comment by mschilli87** [comment]
Just fyi, as a alternative, you could keep ddg as the default and prefix your search with `!g` as this will make ddg forward your search to Google.
